### ISSUE TITLE
[Issue]: Test - Compilation failure on RHEL 8 / ALIOS

### ISSUE THREAD

**Issue by kiritigowda** [ROCm/rpp#454]
## CTest Failure 1
```
Observed error:
In file included from /opt/rocm-6.3.0-14771/share/rpp/test/HOST/../rpp_test_suite_misc.h:25:
1: /opt/rocm-6.3.0-14771/share/rpp/test/HOST/../rpp_test_suite_common.h:1286:48: error: no member named 'setprecision' in namespace 'std'
1:         std::cout << " " << std::fixed << std::setprecision(precision) << static_cast<Rpp32f>(src[i]) << " ";
1:                                           ~~~~~^
1: [ 60%] Linking CXX executable BatchPD_host_pln1
1: In file included from /opt/rocm-6.3.0-14771/share/rpp/test/HOST/Tensor_host.cpp:33:
1: /opt/rocm-6.3.0-14771/share/rpp/test/HOST/../rpp_test_suite_common.h:1286:48: error: no member named 'setprecision' in namespace 'std'
1:         std::cout << " " << std::fixed << std::setprecision(precision) << static_cast<Rpp32f>(src[i]) << " ";
1:                                           ~~~~~^
1: [ 70%] Linking CXX executable BatchPD_host_pln3
1: [ 80%] Linking CXX executable BatchPD_host_pkd3
1: [ 80%] Built target BatchPD_host_pln1
1: [ 80%] Built target BatchPD_host_pln3
1: [ 80%] Built target BatchPD_host_pkd3
1: 1 error generated.
1: make[2]: *** [CMakeFiles/Tensor_misc_host.dir/build.make:76: CMakeFiles/Tensor_misc_host.dir/Tensor_misc_host.cpp.o] Error 1
1: make[1]: *** [CMakeFiles/Makefile2:117: CMakeFiles/Tensor_misc_host.dir/all] Error 2
1: make[1]: *** Waiting for unfinished jobs....
1: 1 error generated.
1: make[2]: *** [CMakeFiles/Tensor_host.dir/build.make:76: CMakeFiles/Tensor_host.dir/Tensor_host.cpp.o] Error 1
1: make[1]: *** [CMakeFiles/Makefile2:91: CMakeFiles/Tensor_host.dir/all] Error 2
1: make: *** [Makefile:91: all] Error 2
```

## Test 2

```
2: In file included from /opt/rocm-6.3.0-14771/share/rpp/test/HIP/Tensor_misc_hip.cpp:25:
2: In file included from /opt/rocm-6.3.0-14771/share/rpp/test/HIP/../rpp_test_suite_misc.h:25:
2: /opt/rocm-6.3.0-14771/share/rpp/test/HIP/../rpp_test_suite_common.h:1286:48: error: no member named 'setprecision' in namespace 'std'
2:  1286 |         std::cout << " " << std::fixed << std::setprecision(precision) << static_cast<Rpp32f>(src[i]) << " ";
2:       |                                           ~~~~~^
2: In file included from /opt/rocm-6.3.0-14771/share/rpp/test/HIP/Tensor_hip.cpp:33:
2: /opt/rocm-6.3.0-14771/share/rpp/test/HIP/../rpp_test_suite_common.h:1286:48: error: no member named 'setprecision' in namespace 'std'
2:  1286 |         std::cout << " " << std::fixed << std::setprecision(precision) << static_cast<Rpp32f>(src[i]) << " ";
2:       |                                           ~~~~~^
2: 1 error generated when compiling for gfx942.
2: failed to execute:/opt/rocm-6.3.0-14771/lib/llvm/bin/clang++  -Xclang -fallow-half-arguments-and-returns -D__HIP_HCC_COMPAT_MODE__=1 --offload-arch=gfx942 -O3 -O3 -Wno-format-nonliteral -parallel-jobs=4   -DUSE_PROF_API=1 -D__HIP_PLATFORM_AMD__=1 -I/opt/rocm-6.3.0-14771/include/hip -I/opt/rocm-6.3.0-14771/include/rpp -isystem /opt/rocm-6.3.0-14771/include -isystem /usr/local/include/opencv4 -fopenmp=libomp -DAUDIO_SUPPORT=1 -DGPU_SUPPORT=1 -DRPP_BACKEND_HIP=1 -std=gnu++17 -x hip -std=gnu++17 -MD -MT CMakeFiles/Tensor_misc_hip.dir/Tensor_misc_hip.cpp.o -MF CMakeFiles/Tensor_misc_hip.dir/Tensor_misc_hip.cpp.o.d -o "CMakeFiles/Tensor_misc_hip.dir/Tensor_misc_hip.cpp.o" -c /opt/rocm-6.3.0-14771/share/rpp/test/HIP/Tensor_misc_hip.cpp
2: make[2]: *** [CMakeFiles/Tensor_misc_hip.dir/build.make:76: CMakeFiles/Tensor_misc_hip.dir/Tensor_misc_hip.cpp.o] Error 1
2: make[1]: *** [CMakeFiles/Makefile2:111: CMakeFiles/Tensor_misc_hip.dir/all] Error 2
2: make[1]: *** Waiting for unfinished jobs....
```

### Operating System

RHEL 8 

### CPU

ALL

### GPU

AMD Instinct MI300

### ROCm Version

ROCm 6.0.0

### ROCm Component

rpp

**Comment by kiritigowda** [comment]
@r-abishek - probably missing a header

**Comment by kiritigowda** [comment]
#453 should fix this issue

**Comment by r-abishek** [comment]
@kiritigowda Addressed by #453
